widget/embedded/src/utils/common.ts

```diff
@@ -29,3 +30,51 @@ export function containsText(text: string, searchText: string): boolean {
   // eslint-disable-next-line @typescript-eslint/no-magic-numbers
   return text.toLowerCase().indexOf(searchText.toLowerCase()) > -1;
 }
+
+export const colorShade = (col: string, amt: number) => {
```

Comment:
Could we change it somehow to have a better background colors for dark theme when blockchain color is near to black?

Reply:
I did not find a solution for this because the color that is coming from the server is almost black